printstraat/backend/app/src/main/resources/bpmn/document-to-printstraat.bpmn

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:camunda="http://camunda.org/schema/1.0/bpmn" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="Definitions_0v9sq8p" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="5.16.0" modeler:executionPlatform="Camunda Platform" modeler:executionPlatformVersion="7.20.0">
+  <bpmn:process id="document-to-printstraat" name="document to printstraat" isExecutable="true" camunda:historyTimeToLive="180">
+    <bpmn:startEvent id="StartEvent_1">
+      <bpmn:outgoing>Flow_0f72yw7</bpmn:outgoing>
+    </bpmn:startEvent>
+    <bpmn:sequenceFlow id="Flow_0f72yw7" sourceRef="StartEvent_1" targetRef="send-document-to-printstraat" />
+    <bpmn:endEvent id="Event_1bv39fx">
+      <bpmn:incoming>Flow_0kgq6hi</bpmn:incoming>
+    </bpmn:endEvent>
+    <bpmn:sequenceFlow id="Flow_0kgq6hi" sourceRef="send-document-to-printstraat" targetRef="Event_1bv39fx" />
```

Comment:
Alle tasks, events etc die een naam hebben en/of iets specifieks doen, graag een passende id geven (zie andere bpmns voor conventie). Daarbij kan je de reguliere pijlen buiten scope houden :)